python/cm/job.py

```diff
@@ -953,14 +953,14 @@ def log_rotation():
             log.info('rotation log from db')
 
     if log_rotation_on_fs:
-        rotation_jobs_on_fs = JobLog.objects.filter(
-            finish_date__lt=timezone.now() - timedelta(days=log_rotation_on_fs)).values('id')
-
-        if rotation_jobs_on_fs:
-
-            for job in rotation_jobs_on_fs:
-                shutil.rmtree(os.path.join(config.RUN_DIR, str(job['id'])))
-            log.info('rotation log from fs')
+        for folder in os.listdir(config.RUN_DIR):
+            if not folder.startswith('.'):
```

Comment:
What this code for? It is not obvious why we have hidden files in RUN_DIR. Please made a comment about.

Reply:
In the RUN_DIR folder is .gitignore

Reply:
It should not be there. Supposed we have to fix this in dockerignore file. Please remove that code and add exception to dockerignore

Reply:
Ok

Reply:
code `if not folder.startswith('.'):` is used for development